src/test/java/use_cases/GetNews/GetNewsInteractorTest.java

```diff
@@ -0,0 +1,50 @@
+package use_cases.GetNews;
+
+import drivers.Finnhub;
+import org.junit.jupiter.api.Test;
+import use_cases.APIAccessInterface;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GetNewsInteractorTest {
+
+    @Test
+    void successTest() {
+        GetNewsInputData inputData = new GetNewsInputData("AAPL");
+        APIAccessInterface driverAPI = new Finnhub();
+
+        // This creates a successPresenter that tests whether the test case is as we expect.
```

Comment:
you can not use the real API here.

this test will ALWAYS fail because we dont push our API keys, so how is our API driver supposed to grab data.

more so, real time data should not be grabbed and tested upon in a testing environment

to resolve, rely on a faux implementation of the APIAccessInterface. anonymous class of APIAccessInterface will be perfect for this

Reply:
you are still using the real API

Reply:
Can you approve now that it actually works?